internal/sdkprovider/service/kafkatopic/kafka_topic.go

```diff
@@ -289,26 +283,39 @@ func resourceKafkaTopicCreate(ctx context.Context, d *schema.ResourceData, m int
 		Tags:        getTags(d),
 	}
 
-	w := &kafkaTopicCreateWaiter{
-		Client:        m.(*aiven.Client),
-		Project:       project,
-		ServiceName:   serviceName,
-		CreateRequest: createRequest,
+	onceCreate.Do(func() {
+		var list []*aiven.KafkaListTopic
+		list, err = client.KafkaTopics.List(project, serviceName)
+		if err != nil {
+			return
+		}
+
+		c.SetV1List(project, serviceName, list)
+	})
```

Comment:
`onceCreate` is in the global scope, so running resources in parallel will race for taking it first. The second to come won't ever get this cached.
In other words, this doesn't work for multiple services.

Reply:
good point, will change that!